docs/admin/reference/backup.rst

```diff
@@ -2,46 +2,66 @@ Backup and Restore
 ==================
 
 Qubes OS has a `backup utility <https://www.qubes-os.org/doc/backup-restore/>`_
-that allows for backup and restoration of user-specified VMs.
+that allows for backup and restoration of user-specified VMs and templates.
+
+SecureDrop Workstation requires only that you back up instance-specific secrets
+and configuration files, although you can optionally back up some additional local
+data.
 
 To perform backups, you will need:
 
- - a LUKS-encrypted external hard drive, with at least 50GB space
+ - a LUKS-encrypted USB or LUKS-encrypted external hard drive (of sufficient size,
```

Comment:
We should link to these docs here: https://workstation.securedrop.org/en/stable/admin/reference/provisioning_usb.html